### ISSUE TITLE
Could not load './es/index.js' from module 'line-intersect'

### ISSUE THREAD

**Issue by thekarel** [psalaets/line-intersect#3]
Seems like `'./es/index.js'` referenced from `package.json` is missing from the published package. 

```
@parcel/core: Failed to resolve 'line-intersect' from './modules/.../x.tsx'
/x.tsx:2:33
> 2 | import {checkIntersection} from 'line-intersect'
>   |                                 ^^^^^^^^^^^^^^^^

@parcel/resolver-default: Could not load './es/index.js' from module 'line-intersect' found in package.json#module
/.../node_modules/line-intersect/package.json:6:13
  5 |   "main": "commonjs/index.js",
> 6 |   "module": "es/index.js",
>   |             ^^^^^^^^^^^^^ './es/index.js' does not exist, did you mean './src/index.js'?'
  7 |   "scripts": {
  8 |     "test": "BABEL_ENV=test mocha --require @babel/register",
```

**Comment by psalaets** [comment]
Thank you for reporting this.

I published version 2.2.4 which should correct this issue.

**Comment by thekarel** [comment]
Thanks for the quick fix @psalaets!
